File: Python/linearsearch.py
def linearsearch(arr):
    cnt=0
    pos=[]
    key=int(input("Enter the Element to be searched for :"))
    for i in range(0, len(arr)):
        if arr[i]==key:
            cnt+=1
            pos.append(i)
    if cnt == 0:
        print(f"{key} not found")
    else:
        print(f"{key} found for {cnt} times at {pos}")

n=10
arr=[]
while(n!=0):
    arr.append(int(input("Enter Element :")))
    n=n-1
print(arr)
linearsearch(arr)

File: C code DSA Algo/linearsearch.c
#include<stdio.h>
#include<stdlib.h>
void main(){
    int i,n,a[20],x;
    printf("Enter the number of elements of the array : \n");
    scanf("%d", &n);
    for(i=0; i<n; i++){
        printf("Enter the %d element of the array :\n", i);
        scanf("%d", &a[i]);
    }
    printf("Enter the element to be searched : \n");
    scanf("%d", &x);
    for(i=0; i<n; i++){
        if(a[i] == x){
            printf("The %d element found at %d position. ", x,i);
        }
        
    }
}